service/raft/raft_group0.cc

```diff
@@ -485,7 +485,7 @@ future<> raft_group0::join_group0(std::vector<gms::inet_address> seeds, shared_p
         }
 
         // TODO: aborts?
-        if (co_await handshaker->post_server_start(g0_info)) {
+        if (co_await handshaker->post_server_start(g0_info, _abort_source)) {
```

Comment:
nit: the title of the commit `group 0: group0_handshaker: add a new parameter to post_server_start ` is kinda weird, we actually care about the fact that this is an `abort_source` parameter, not just "a" parameter :P

Reply:
I'll change it. I have to restart CI anyway.